src/stories/charts/donut.stories.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+// Libraries
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import chaptersAddon from 'react-storybook-addon-chapters';
+import { withKnobs, array } from '@storybook/addon-knobs';
+import { Router, Route, IndexRoute, BrowserRouter} from 'react-router-dom'
+
+// Styling
+import '../../stories.scss';
+
+// Component
+import { Donut } from '@red-hat-insights/insights-frontend-components';
+
+// README
+import DonutReadme from '../../docs/charts/donut/README.md';
+import { withReadme }  from 'storybook-readme';
+
+storiesOf('Charts', module)
+    .addDecorator(withReadme(DonutReadme))
+    .addDecorator(withKnobs)
+    .addWithChapters(
+        'Donut',
+        {
+            chapters: [
+                {
+                    title: 'Donut',
+                    sections: [
+                        {
+                            title: 'Preview',
+                            sectionFn: ('', () => {
+                                let DonutValues = [
+                                    ['value1', 5],
+                                    ['value2', 10],
+                                    ['value3', 20],
+                                    ['value4', 40]
+                                ];
+
+                                return(
+                                    <Donut values={ DonutValues } totalLabel='issues' identifier='storybook-donut'/>
```

Comment:
It would be really awesome if consumers could test this with knobs
```JSX
<Donut values={ [
    ['value1', input('value 1', 5)],
    ['value2', input('value 1', 10)],
    ['value3', input('value 1', 20)],
    ['value4', input('value 1', 40)]
  ] }
  totalLabel='issues'
  identifier='storybook-donut'
/>
```

Reply:
Unfortunately, the chart is static once it loads the data and cannot be manipulated unless the chat is redrawn. I can see if I can find a solution to be able to do this.

Reply:
Sure thing! If you want you can merge this one, since the code looks fine

Reply:
👍